core/templates/pages/blog-dashboard-page/blog-dashboard-tile/blog-dashboard-tile.component.html

```diff
@@ -0,0 +1,85 @@
+<mat-card class="blog-dashboard-tile">
+  <div class="blog-dashboard-tile-content">
+    <mat-card-content>
+      <div class="blog-dashboard-tile-header">
+        <mat-card-title class="blog-dashboard-tile-title">{{blogPostSummary.title}}</mat-card-title>
+        <button mat-icon-button>
+          <mat-icon class="align-text-bottom pl-4">more_vert</mat-icon>
+        </button>
+      </div>
+      <mat-card-subtitle class="blog-dashboard-tile-subtitle">
+        {{getDateStringInWords(blogPostSummary.publishedOn)}}
```

Comment:
Save this in a variable (during ngOnInit) and use that variable instead.

Reply:
Done. (here and in blog-card.component )